content/information-distribution/signature.md

```diff
@@ -17,14 +17,26 @@ Informo signed events are built using a common structure, inspired from [Matrix
 `m.room.encrypted`
 events](https://matrix.org/docs/spec/client_server/r0.4.0.html#m-room-encrypted):
 
-|    Field     |   Type   |                                    Description                                    |
-| ------------ | -------- | --------------------------------------------------------------------------------- |
-| `algorithm`  | `string` | Algorithm used for signing the content. 🔧 Define allowed algorithms               |
-| `sender_key` | `string` | The public part of the key used for signing the event                             |
-| `device_id`  | `string` | 🔧 Optional, ID of the sending device, may be used for Megolm                      |
-| `session_id` | `string` | 🔧 Optional, ID of the session used to encrypt the message, may be used for Megolm |
-| `signature`  | `string` | Generated signature                                                               |
-| `signed`     | `object` | Content that has been signed                                                      |
+|    Field     |      Type      |                                    Description                                     |
+|--------------|----------------|------------------------------------------------------------------------------------|
+| `algorithm`  | `string`       | Algorithm used for signing the content. 🔧 Define allowed algorithms               |
+| `sender_key` | `string`       | The public part of the key used for signing the event                              |
+| `device_id`  | `string`       | 🔧 Optional, ID of the sending device, may be used for Megolm                      |
+| `session_id` | `string`       | 🔧 Optional, ID of the session used to encrypt the message, may be used for Megolm |
+| `signature`  | `string`       | Generated signature                                                                |
+| `signed`     | `SignedObject` | Object to be signed                                                                |
+
+Where:
+
+* `SignedObject` is a map using the following structure:
+
+| Parameter          |       Type       | Req. |              Description               |
+|:-------------------|:-----------------|:----:|:---------------------------------------|
+| `sender`           | `string`         |  x   | The matrix user ID of the sender.      |
+| `content`          | `object`         |  x   | The content of the event to be signed. |
+
+If the value of the `signed.sender` field does not match the original matrix
```

Comment:
I find this sentence also ambiguous about which event is being talked about. Also, please pay attention to capitalising the first letter of names such as Matrix.